<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddRecipients</class>
 <widget class="QDialog" name="AddRecipients">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>880</width>
    <height>480</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
color: #07142A;
font-family: Roboto, Helvetica;
font-size: 14px;
}
#leftPane, #rightPane {
background-color: #ffffff;
}
#leftPane {
border-right: 1px solid #E7EAEF;
}</string>
  </property>
  <layout class="QGridLayout" name="AddRecipientsLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="ItemList" name="leftPane"/>
   </item>
   <item row="0" column="1">
    <widget class="ItemList" name="rightPane"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QWidget" name="navigationArea" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>48</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#navigationArea {
border-top: 1px solid #E7EAEF;
background-color: #F3F5F7;
}
QPushButton {
padding: 10px 12px;
border: 1px solid transparent;
border-radius: 4px;
color: #2F70B6;
font-size: 12px;
font-weight: 700;
}
QPushButton:hover {
background-color: #EAF1F8;
}
QPushButton:pressed {
background-color: #BFD3E8;
}
QPushButton:disabled {
color: #82A9D3;
}
QPushButton:default {
color: #ffffff;
border-color: #2F70B6;
background-color: #2F70B6;
}
QPushButton:default:hover {
border-color: #2B66A6;
background-color: #2B66A6;
}
QPushButton:default:pressed {
border-color: #215081;
background-color: #215081;
}
QPushButton:default:disabled {
border-color: #82A9D3;
background-color: #82A9D3;
}</string>
     </property>
     <layout class="QHBoxLayout" name="navigationAreaLayout" stretch="0,0,0,1,0">
      <property name="spacing">
       <number>16</number>
      </property>
      <property name="leftMargin">
       <number>24</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>24</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="fromCard">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Certificate from card</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="fromFile">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Certificate from file</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="fromHistory">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Latest used certificate</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="actionLayout">
        <property name="spacing">
         <number>16</number>
        </property>
        <item>
         <widget class="QPushButton" name="cancel">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
border: 1px solid #AD2A45;
color: #AD2A45;
}
QPushButton:hover {
background-color: #F5EBED;
}
QPushButton:pressed {
background-color: #E1C1C6;
}</string>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="confirm">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Confirm</string>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ItemList</class>
   <extends>QScrollArea</extends>
   <header>widgets/ItemList.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
